NetMQ.ReactiveExtensions/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {    
+    "NetMQ": "4.0.0-rc5",
+    "protobuf-net": "2.1.0",
+    "System.Reactive.Core": "3.1.0"
+  },
+
+  "frameworks": {
+    "netcoreapp1.0": {
```

Comment:
Apparently `netcoreapp*` is for executable programs. As this is a library, it should only target `netstandard*` as well as a full framework such as `net45`. My understanding is that `netcoreapp1.0` is compatible with `netstandard1.6`.